docker/1.6.0/py2/Dockerfile.gpu

```diff
@@ -42,6 +47,67 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && ln -s $(which python) /usr/local/bin/python
 
+# Install cuda-dev libraries as a dependency for Horovod with MXNet backend
+ENV CUDA_LIB_URL=https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64
+RUN wget ${CUDA_LIB_URL}/libcublas10_10.2.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/libcublas-dev_10.2.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-curand-dev-10-1_10.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-cusolver-dev-10-1_10.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-nvrtc-10-1_10.1.243-1_amd64.deb \
+    ${CUDA_LIB_URL}/cuda-nvrtc-dev-10-1_10.1.243-1_amd64.deb \
+ && dpkg -i libcublas10_10.2.1.243-1_amd64.deb \
+    libcublas-dev_10.2.1.243-1_amd64.deb \
+    cuda-curand-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-cusolver-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-dev-10-1_10.1.243-1_amd64.deb \
+ && apt-get install -f \
+ && rm libcublas10_10.2.1.243-1_amd64.deb \
+    libcublas-dev_10.2.1.243-1_amd64.deb \
+    cuda-curand-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-cusolver-dev-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-10-1_10.1.243-1_amd64.deb \
+    cuda-nvrtc-dev-10-1_10.1.243-1_amd64.deb
+
+###########################################################################
+# Horovod dependencies
+###########################################################################
+
+# Install Open MPI
+RUN mkdir /tmp/openmpi \
+ && cd /tmp/openmpi \
+ && curl -fSsL -O https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.1.tar.gz \
+ && tar zxf openmpi-4.0.1.tar.gz \
+ && cd openmpi-4.0.1 \
+ && ./configure --enable-orterun-prefix-by-default \
+ && make -j $(nproc) all \
+ && make install \
+ && ldconfig \
+ && rm -rf /tmp/openmpi
+
+RUN apt-get update \
+ && apt-get install -y \
+    python \
+    python-pip
+
+# Create a wrapper for OpenMPI to allow running as root by default
+RUN mv /usr/local/bin/mpirun /usr/local/bin/mpirun.real \
+ && echo '#!/bin/bash' > /usr/local/bin/mpirun \
+ && echo 'mpirun.real --allow-run-as-root "$@"' >> /usr/local/bin/mpirun \
+ && chmod a+x /usr/local/bin/mpirun
+
+# Configure OpenMPI to run good defaults:
+#   --bind-to none --map-by slot --mca btl_tcp_if_exclude lo,docker0
+RUN echo "hwloc_base_binding_policy = none" >> /usr/local/etc/openmpi-mca-params.conf \
+ && echo "rmaps_base_mapping_policy = slot" >> /usr/local/etc/openmpi-mca-params.conf
+
+# Set default NCCL parameters
+RUN echo NCCL_DEBUG=INFO >> /etc/nccl.conf
+
+ENV LD_LIBRARY_PATH=/usr/local/openmpi/lib:$LD_LIBRARY_PATH
+ENV PATH /usr/local/openmpi/bin/:$PATH
+ENV PATH=/usr/local/nvidia/bin:$PATH
```

Comment:
we should be consistent in how we assign. Since we are only assigning one variable in each line, you can omit the "="

see https://docs.docker.com/engine/reference/builder/#env

Reply:
I think it's better to have "=" rather than " ". But yes, it needs to be consistent.

Reply:
Sure, to maintain consistency w/ Dockerfile you can keep the "=" implementation, but let's just make sure it's consistent

Reply:
I think @arjkesh point is correct. If the `ENV` is of single line then we don't need `=` otherwise requires `=` according to official docker rule. Let's follow the official documentation.

Reply:
Docker does not specify either as best practice. According to the convention set by the rest of our dockerfiles, both on this repository and other framework repositories, we have been using "=" for ENV declarations.

Reply:
Okay then all the `ENV` should have `=` right ? @saimidu

Reply:
Yes. Please use the "=" convention.

Reply:
Done